test/unit/commands/performance-specs.js

```diff
@@ -0,0 +1,39 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import sinon from 'sinon';
+import Bootstrap from 'appium-android-bootstrap';
+import AndroidDriver from '../../..';
+
+let driver;
+let sandbox = sinon.sandbox.create();
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('getperformancedata', () => {
```

Comment:
You have this in the unit tests, but I don't see you stubbing out the methods, which makes me think this is actually still running adb commands like an e2e test